docs/UI-UX/recommended-technologies.md

```diff
@@ -11,40 +44,395 @@ Additionally, while some of these technologies are presented as alternate option
+- [Vite HMR](https://vitejs.dev/guide/features.html#hot-module-replacement)
+- [Vite Configuration](https://vitejs.dev/config/)
+- [Vite Plugins](https://vitejs.dev/plugins/)
+- [Vite Troubleshooting](https://vitejs.dev/guide/troubleshooting.html)
+
+From [Scaffolding your First Vite Project](https://vitejs.dev/guide/#scaffolding-your-first-vite-project)
+
+##### Installing Vite
+
+```sh
+# Install Vite
+npm install -D vite
+```
+##### Bootstrapping a project
+```sh
+npm create vite@latest
+```
+This will start a CLI interface that will create multiple types of projects that Vite supports: React, Angular, Svelte, Solid, Vue, or even Vanilla HTML and JavaScript.
 
-### Bootstrapping
+![Vite Wizard](./images/vite-wizard.png "Vite Wizard")
 
-There are many different ways to bootstrap a React application. Two great tool sets to use are [create-react-app](https://create-react-app.dev/) and [vite](https://vitejs.dev/guide).
 
-#### create-react-app
+## React
+
+React is a framework developed and maintained by Meta (Formerly Facebook). React is used throughout Microsoft's product stack and has a vast open source community.
+
+### Quick note on create-react-app
+
+> __create-react-app is deprecated as of January 2023 and should be avoided for new projects.__ It is recommended to use more modern tools like [Vite](#vite) for better performance and flexibility. For more details, refer to the [official React blog](https://react.dev/blog) and [Vite documentation](https://vitejs.dev/guide/why.html).
+
 
 From [Adding TypeScript](https://create-react-app.dev/docs/adding-typescript/)
 
 ```sh
 npx create-react-app my-app --template typescript
 ```
 
-#### Vite
+### Documentation & Using older React coding patterns
 
-From [Scaffolding your First Vite Project](https://vitejs.dev/guide/#scaffolding-your-first-vite-project)
+One can expect to find a multitude of guides, answers, and posts on how to work with React; don't take everything at face value. The best place to review React concepts is the React documentation. From there, you can review articles from various sources such as [React Community Articles](https://reactjs.org/community/articles.html), [Kent C Dodd's Blog](https://kentcdodds.com/blog?q=react), [CSS Tricks Articles](https://css-tricks.com/?s=react), and [Awesome React](https://github.com/enaqx/awesome-react).
 
-```sh
-# npm 6.x
-npm init vite@latest my-app --template react-ts
+The React API has changed dramatically over time. Older resources may contain solutions or patterns that have since been changed and improved upon. Modern React development uses the [React Hooks](https://reactjs.org/docs/hooks-intro.html) pattern. Rarely will you have to implement something using [React Class](https://reactjs.org/docs/react-component.html) pattern. If you're reading an article/answer/docs that instruct you to use the class pattern you may be looking at an out-of-date resource.
+
+
+## Best Practices to Organize Your React Codebase
```

Comment:
I wonder if this is the correct place for this... or should we move this into a react section with link to this.

Reply:
Not sure it merits its own separate page... I see your point but I think this section can still fit into where it is right now